angular-hotkeys/angular-hotkeys.d.ts

```diff
@@ -9,13 +9,16 @@ declare module ng.hotkeys {
 
     interface HotkeysProvider {
         template: string;
+        templateTitle:string;
         includeCheatSheet: boolean;
         cheatSheetHotkey: string;
         cheatSheetDescription: string;
 
-        add(combo: string, description: string, callback: (event: Event, hotkeys: ng.hotkeys.Hotkey) => void): void;
```

Comment:
we should keep the old APIs around since this change will break back compat for existing projects.